### PR TITLE
logo n update edit class ug

### PR DESCRIPTION
**Pull request by softmagnet**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -221,12 +223,15 @@ editclass 1 [cn/CLASS_NAME] [ct/CLASS_TIMING] [r/RATE] [l/LOCATION]
 ```
 
 * Edits the class at the specified `INDEX`. The index refers to the index number shown in the displayed class
-  list in the `classes` tab. The index **must be a positive integer** 1, 2, 3, …​
+  list in the `classes` tab. 
+  * The index **must be a positive integer** 1, 2, 3, …​
+  * The index must also not be out of rage.
```
Comment (Ongzl) [inline docs/UserGuide.md:228]: spelling error range.

Comment (Ongzl) [reply docs/UserGuide.md:228]: maybe can elaborate what the range means also

File: docs/UserGuide.md [hunk 2]
```diff
@@ -221,12 +223,15 @@ editclass 1 [cn/CLASS_NAME] [ct/CLASS_TIMING] [r/RATE] [l/LOCATION]
...
 * At least one of the optional fields must be provided.
 * `CLASS_TIMING` can only start at the hour mark or half hour mark.
+* Edit commands that will create a **clash of class timing** is not accepted.
```
Comment (Ongzl) [inline docs/UserGuide.md:231]: maybe can specify as clash of timing with other existing classes?

**Review by Ongzl — COMMENTED** [review]
otherwise LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (s7u4rt99) [inline docs/UserGuide.md:231]: Maybe u can change it to
```suggestion
* Edit commands that will create a **clash of `CLASS_TIMING`** is not accepted.
```

Comment (s7u4rt99) [reply docs/UserGuide.md:231]: Other than this, LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (softmagnet) [reply docs/UserGuide.md:228]: Ok i add in last part of notes about command format

File: docs/UserGuide.md (hunk 2 above)
Comment (softmagnet) [reply docs/UserGuide.md:231]: ok done.

**Pull request merged**